src/lib/sd-jwt.ts

```diff
@@ -57,3 +57,30 @@ export function decodeSdJwt(encodedJwt: string) {
 		return undefined;
 	}
 }
+
+export default crypto;
+export function provideHasher(alg: string) {
+	let browserAlg: string = '';
+	switch (alg.toLowerCase()) {
+		case 'sha-256':
+			browserAlg = "SHA-256"
+			break;
+		case 'sha-384':
+			browserAlg = "SHA-384"
+			break;
+		case 'sha-512':
+			browserAlg = "SHA-512"
+			break;
+	}
+	var enc = new TextEncoder();
+	const hasherAndAlgorithm: HasherAndAlgorithm = {
+		// TODO: how do you properly cast this?
```

Comment:
This imho is the correct way to do this. Uint8Array is a different format than ArrayBuffer, so conversion needs to be done